src/components/Identity/configValidators.js

```diff
@@ -14,7 +14,8 @@ import { boolean } from "../../utils/validation";
 
 const configValidators = {
   thirdPartyCookiesEnabled: boolean().default(true),
-  idMigrationEnabled: boolean().default(true)
+  idMigrationEnabled: boolean().default(true),
+  idOverwriteEnabled: boolean().default(true)
```

Comment:
Do you think true is the right default?

Reply:
I think false might be a safer bet since that’s how it behaves today in Visitor.js

Reply:
Let’s ask Mitch though